contrib/aws/aws-sdk-go/aws/example_test.go

```diff
@@ -25,3 +31,33 @@ func Example() {
 		Bucket: aws.String("some-bucket-name"),
 	})
 }
+
+func TestWithErrorCheck(t *testing.T) {
+	testOpts := func(errExist bool, opts ...awstrace.Option) func(t *testing.T) {
+		return func(t *testing.T) {
+			mt := mocktracer.Start()
+			defer mt.Stop()
+
+			cfg := aws.NewConfig().WithRegion("us-west-2")
+			sess := session.Must(session.NewSession(cfg))
+			sess = awstrace.WrapSession(sess, opts...)
+
+			s3api := s3.New(sess)
+			s3api.CreateBucket(&s3.CreateBucketInput{
+				Bucket: aws.String("some-bucket-name"),
+			})
+
+			spans := mt.FinishedSpans()
+			assert.True(t, len(spans) > 0)
```

Comment:
One has to create a span before hand, an example of that can be found in other tests of `aws_test.go`.
```go
root, ctx := tracer.StartSpanFromContext(context.Background(), "test")
sess := session.Must(session.NewSession(aws.NewConfig().WithRegion("us-west-2")))
sess = WrapSession(sess, opts...)
s3api := s3.New(sess)
s3api.CreateBucketWithContext(ctx, &s3.CreateBucketInput{
	Bucket: aws.String("some-bucket-name"),
})
root.Finish()
```

Reply:
I've added the creation of the span.